zipline/utils/events.py

```diff
@@ -169,14 +169,35 @@ def _build_time(time, kwargs):
         return datetime.time(**kwargs)
 
 
-class EventManager(object):
+@object.__new__
+class _nop_context(object):
```

Comment:
I wasn't sure if we used contextlib32 in zipline. This is mainly for clarity though, we never want the features of exitstack.